src/include/sof/math/trig.h

```diff
@@ -50,12 +63,12 @@ void cordic_sin_cos(int32_t th_rad_fxp, cordic_cfg type, int32_t *sign, int32_t
  */
 static inline int32_t sin_fixed_32b(int32_t th_rad_fxp)
 {
+	int32_t th_cdc_fxp;
 	int32_t sign;
 	int32_t b_yn;
 	int32_t xn;
-	int32_t th_cdc_fxp;
 	cordic_cfg type = EN_32B_CORDIC_SINE;
-	cordic_sin_cos(th_rad_fxp, type, &sign, &b_yn, &xn, &th_cdc_fxp);
+	cordic_approx(th_rad_fxp, type, &sign, &b_yn, &xn, &th_cdc_fxp, cint32_t *cis);
```

Comment:
Does this even compile?

Reply:
I was wondering cis too, where is it defined. It's a good idea to create pull request first as draft.

Reply:
I am sorry, my  sincere apologies . I will fix the compilation issues.